packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -48,25 +51,24 @@ const Drawer = (props: THeaderProps) => {
   )
 }
 
-const DrawerTools = (props: TNavEditions) => (
+const DrawerTools = (editions: TNavEditions) => (
   <div className="o-header__drawer-tools">
     <button
       type="button"
       className="o-header__drawer-tools-close"
       title="Close side navigation menu"
       aria-controls="o-header-drawer"
-      data-trackable="close"
-    >
+      data-trackable="close">
       <span className="o-header__visually-hidden">Close side navigation menu</span>
     </button>
     <a className="o-header__drawer-tools-logo" href="/" data-trackable="logo">
       <span className="o-header__visually-hidden">Financial Times</span>
     </a>
-    {props.current && <p className="o-header__drawer-current-edition">{`${props.current.name} Edition`}</p>}
+    {editions && <EditionsSwitcher {...editions} />}
```

Comment:
**suggestion(non-blocking):** I like in Origami that it is explicit about passing the current and other editions which is easier to read. We should keep these templates as close to the Origami template as possible. But we have plans to move away from this template in the future (probably in Q1 2025) so no need to do this.